composer.json

```diff
@@ -1,37 +1,57 @@
 {
-    "name": "codeclimate/php-test-reporter",
-    "description": "PHP client for reporting test coverage to Code Climate",
```

Comment:
The reformatting, yes, but not the additions at the bottom.
Sorry, My PhpStorm settings differ from the previous indention.